zotero_master/translators/PicaSWB_batch.js

```diff
@@ -279,7 +279,7 @@ function WriteItems() {
 		}
 		else {
 			var elementString = cleanElement.join("");
-			elementString = elementString.replace(/\\n/g, '\n');
+			elementString = elementString.replace(/\\n/g, '\n').replace(/\\"/g, '"');

```

Comment:
Liebe @helenanebel, unescape-Zeichen `\` bitte nur im Sigle entfernen, also wenn nur ein Eintrag exportiert wird. Im Batch muss das Anführungszeichen jedoch maskiert werden, da sonst eine Fehlermeldung beim Hochladen in WiniBW erscheint.

Reply:
das war ein Irrtum von mir. Alles passt.